config/config.go

```diff
@@ -240,8 +240,18 @@ type TiKVClient struct {
 	GrpcKeepAliveTimeout uint `toml:"grpc-keepalive-timeout" json:"grpc-keepalive-timeout"`
 	// CommitTimeout is the max time which command 'commit' will wait.
 	CommitTimeout string `toml:"commit-timeout" json:"commit-timeout"`
+
 	// MaxTxnTimeUse is the max time a Txn may use (in seconds) from its startTS to commitTS.
 	MaxTxnTimeUse uint `toml:"max-txn-time-use" json:"max-txn-time-use"`
+
+	// MaxBatchSize is the max batch size when calling batch commands API.
+	MaxBatchSize uint `toml:"max-batch-size" json:"max-batch-size"`
+	// If TiKV load is greater than this, TiDB will wait for a while to avoid little batch.
+	TiKVHeavyLoadToBatch uint `toml:"tikv-heavy-load-to-batch" json:"tikv-heavy-load-to-batch"`
```

Comment:
em, I think we can remove TiKV here.

Reply:
How about `StorageEngineOverloadThreshold`, or just `OverloadThreshold`?

Reply:
I prefer OverloadThreshold